src/test/java/org/openelisglobal/systemUserSection/SystemUserSectionServiceTest.java

```diff
@@ -0,0 +1,66 @@
+package org.openelisglobal.systemUserSection;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.openelisglobal.BaseWebContextSensitiveTest;
+import org.openelisglobal.common.util.ConfigurationProperties;
+import org.openelisglobal.systemusersection.service.SystemUserSectionService;
+import org.openelisglobal.systemusersection.valueholder.SystemUserSection;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class SystemUserSectionServiceTest extends BaseWebContextSensitiveTest {
+
+    @Autowired
+    private SystemUserSectionService systemUserSectionService;
+
+    @Before
+    public void setUp() throws Exception {
+        executeDataSetWithStateManagement("testdata/system-user-section.xml");
+
+    }
+
+    @Test
+    public void testDataBase() {
+        List<SystemUserSection> systemUserSections = systemUserSectionService.getAll();
+        systemUserSections.forEach(systemUserSection -> {
```

Comment:
@Isabirye1515 we no longer need to pring this on the console to avoid logs clutter